### DISCUSSION TITLE
A confusing bug

### DISCUSSION THREAD

**Discussion by rookieThresh** [krvajal/vscode-fortran-support#941] [Q&A - deprecated see Fortran Discourse] [answered]
It's a easy example.But I can't get the right result.
<img width="497" alt="1" src="https://github.com/fortran-lang/vscode-fortran-support/assets/125661029/6679e8f5-5067-45c7-b2c7-f82a451b6578">
<img width="437" alt="2" src="https://github.com/fortran-lang/vscode-fortran-support/assets/125661029/cae47d3e-3071-4b53-a3c5-181e1f20c77d">

**Comment by gnikit** [comment, marked as answer]
Please redirect questions about the Fortran programming language either to
- Fortran-lang's Discourse: https://fortran-lang.discourse.group/
- StackOverflow: https://stackoverflow.com/
- comp.lang.fortran (Google Group): https://groups.google.com/g/comp.lang.fortran?pli=1

The Discussions here are for Discussions for the extension's development.

**Reply by rookieThresh** [reply]
ok,thanks
